### ISSUE TITLE
Docs: Duplicate commas when describing default compiler options

### ISSUE THREAD

**Issue by bennycode** [microsoft/TypeScript-Website#2520]
**Page URL:** https://www.typescriptlang.org/tsconfig

**Issue:** There are two commas when a config has a default value.

**Examples:**

- https://www.typescriptlang.org/tsconfig#useUnknownInCatchVariables
![image](https://user-images.githubusercontent.com/469989/188459962-f554a2fd-f8e8-435b-8750-ef6183d9ce0b.png)

- https://www.typescriptlang.org/tsconfig#module
![image](https://user-images.githubusercontent.com/469989/188460200-8c6e7cd9-e7c7-4dde-959e-ead0cfb7400b.png)

**Recommended Fix:** Remove one of the commas that separates the default value from the available options.

**Comment by justinyaodu** [comment]
I think the issue happens because [`defaultsForOptions` has some values that are lists of strings](https://github.com/microsoft/TypeScript-Website/blob/v2/packages/tsconfig-reference/scripts/tsconfigRules.ts#L196), and [when these lists are converted into strings](https://github.com/microsoft/TypeScript-Website/blob/v2/packages/tsconfig-reference/scripts/tsconfig/generateMarkdown.ts#L254), an extra comma is added.

For example, the following code...
```js
console.log(String(["`CommonJS` if [`target`](#target) is `ES3` or `ES5`,", "`ES6`/`ES2015` otherwise."]))
```
...has this output with an extra comma:
```
`CommonJS` if [`target`](#target) is `ES3` or `ES5`,,`ES6`/`ES2015` otherwise.
```

**Comment by agilgur5** [comment]
Looks like this was fixed in #2288 ([this line](https://github.com/microsoft/TypeScript-Website/pull/2288/files#diff-aa29228736a785d50641fac6518b5025b866d3be49178c2ee53e54147a6d7e07L270))

**Comment by agilgur5** [comment]
Looks like this was fixed at some point -- [current site](https://www.typescriptlang.org/tsconfig#module) no longer has these duplicate commas:
![Screenshot 2023-07-18 at 3 58 14 PM](https://github.com/microsoft/TypeScript-Website/assets/4970083/4434362e-0180-4850-bc1c-1caa9ede6b3a)
